drivers/charger/charger_handlers.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/syscall_handler.h>
+#include <zephyr/drivers/charger.h>
+
+static inline int z_vrfy_charger_get_prop(const struct device *dev, const charger_prop_t prop,
+					  union charger_propval *val)
+{
+	union charger_propval k_val = {0};
```

Comment:
no need to initialize this one, `z_impl_charger_get_prop` is going to set it

Reply:
ack